Automattic/wp-calypso
# Admin bar: Remove Reader tooltip

Reverts https://github.com/Automattic/wp-calypso/pull/93972
Reverts https://github.com/Automattic/wp-calypso/pull/94157

## Proposed Changes

Removes the tooltip announcing the new position of the Reader menu in the admin bar that was introduced in https://github.com/Automattic/wp-calypso/pull/93972 since at this time is probably pointless for infrequent users and confusing for new users.

Before | After
--- | ---
<img width="567" alt="Screenshot 2024-10-07 at 13 19 05" src="https://github.com/user-attachments/assets/1ed84732-db91-46b1-b368-349438ab75a7"> | <img width="558" alt="Screenshot 2024-10-07 at 13 26 23" src="https://github.com/user-attachments/assets/e53e09bc-4a43-4e18-807c-c5200e5afb8a">
 

## Why are these changes being made?

To clean up unnecessary code.

## Testing Instructions

- Use the Calypso live link below
- Clear the `dismissible-card-masterbar-reader-popover` preference in the environment tools:

<img width="767" alt="Screenshot 2024-10-07 at 13 18 51" src="https://github.com/user-attachments/assets/e4376b32-7f6a-4b33-ae7e-9a4d7615b240">

- Make sure the Reader tooltip does not show up

## Pre-merge Checklist
